modules/appeals_api/spec/services/appeals_api/remove_pii_spec.rb

```diff
@@ -3,8 +3,158 @@
 require 'rails_helper'
 require AppealsApi::Engine.root.join('spec', 'spec_helper.rb')
 
+def update_appeal_status(appeal, status, code: nil, detail: nil)
+  # At the time of writing, the `update_status` method for each appeal model involves kicking off a sidekiq job to
+  # create a matching StatusUpdate record. This is unwieldy in tests, so this method approximates the
+  # `appeal.update_status!` method without involving sidekiq.
```

Comment:
This note relates to [the issue I described in Slack here](https://lighthouseva.slack.com/archives/GPX76BVKN/p1712104410851629). The way that `update_status` is written on our appeal models, a sidekiq job would have to be kicked off each time a status update is made. I don't think that's a good pattern (and I don't think this workaround is very good either), but this helper method seemed like the shortest path to getting this done while we're all strapped for time. Ultimately I think it would be better to stop kicking off sidekiq jobs for creation of status updates.

Reply:
Thanks for documenting this, and I agree with your approach and the desired end state.